stingray/events.py

```diff
@@ -19,8 +19,8 @@
 
 
 class EventList(object):
-    def __init__(self, time=None, energy=None, ncounts=None, mjdref=0, dt=0, notes="",
-            gti=None, pi=None):
+    def __init__(self, time=None, energy=None, ncounts=None, mjdref=0, dt=0,
+                 notes="", gti=None, pi=None):
```

Comment:
What does “notes” do? If only there was a docstring to tell me ...😉

Reply:
Right. I have no idea O_o

Reply:
Done :D

Reply:
:+1: